di_website/spotlight/models.py

```diff
@@ -18,13 +22,23 @@ class Meta():
     country_code = models.CharField(max_length=100, help_text='e.g. UG, KE', default='')
     country_name = models.CharField(max_length=255)
     currency_code = models.CharField(max_length=100, help_text='UGX, KES', default='')
-
+    datasources_description = models.TextField(help_text='A description for data sources section', default='')
+    datasources_links = StreamField(
+        DatasourcesStreamBlock,
+        verbose_name="Datasources Links",
+        null=True,
+        blank=True
+    )
```

Comment:
Won't need the StreamBlock
`datasource_links = StreamField([
        ('link', LinkBlock()),
    ], null=True, blank=True)`

Reply:
Updated `datasource_links` to use this inline streamfield